src/index.js

```diff
@@ -1,37 +1,103 @@
-//import Forecast from '@models/Forecast.js';
-
+import { API_URL, API_KEY } from '@models/constants.js';
+import Forecast from '@models/Forecast.js';
 import 'normalize.css';
 import "@/styles/styles.scss";
-
 import axios from 'axios';
 
-window.searchForecast = () => {
-  const input = document.querySelector('.search__input');
-  const autocomplete = new google.maps.places.Autocomplete(input);
-  autocomplete.addListener("place_changed", () => {
-    const place = autocomplete.getPlace();
+class App {
+  constructor() {
+    this.elements = {
+      input: document.querySelector('.search__input'),
+      clearButton: document.querySelector('.search__input-delete')
+    };
+
+    this.forecast = new Forecast();
+
+    this.service = new google.maps.places.PlacesService(this.elements.input);
+    this.autocomplete = new google.maps.places.Autocomplete(this.elements.input);
+
+    this.searchUserGeolocation();
+    this.addListeners();
+  }
+
+  searchUserGeolocation() {
+    navigator.geolocation.getCurrentPosition(
+      place => {
+        this.successSearch(place);
+      }, () => {
+        this.elements.input.value = 'Kyiv, Ukraine';
+        this.searchFromQuery();
+      });
+  }
+
+  successSearch(place) {
+    const location = {
+      lat: place.coords.latitude,
+      lon: place.coords.longitude
+    }
+
+    this.elements.input.value = "now this is your location";
```

Comment:
Looks very strange.
I think It's better to display city name somewhere (not inside input value). You can get it from your weather api response
`data.city_name`